add a link to failed DCO status to info about DCO

robbkidd opened the pull request:
Make a failed DCO status include a link to our DCO information in the chef/chef CONTRIBUTING doc. Is chef/chef a weird thing to link to from this habitat-sh project?

Also, bonus removal of trailing whitespace.

(First commit intentionally not signed-off, but dcob is not currently monitoring dcob it seems. 😄 )

robbkidd committed:
add a link to failed DCO status to info about DCO

thommay reviewed (changes requested):
Target URL should be a config option…

robbkidd commented:
![can do](https://cloud.githubusercontent.com/assets/517302/19069727/0f15d9e4-89f6-11e6-8b67-a6a6b942af3a.gif)

robbkidd committed:
make the DCO status check target_url configurable

The most sensible default is probably just a link to the
certificate itself. Projects using this bot should probably
override that with a link to project specific information
and instructions for signing-off commits.

Signed-off-by: Robb Kidd <robb@thekidds.org>

robbkidd commented:
This configurable DCO info link works when paired with #3's fixes to `default.toml` and plan.

robbkidd commented:
Rebased to master.

Hmm. The new Review system doesn't seem to like rebases.

![error](https://cloud.githubusercontent.com/assets/517302/19116882/fa5da76c-8ae4-11e6-9276-0f0ce52b5cd7.gif)
